include/deal.II/grid/grid_tools.h

```diff
@@ -555,7 +555,22 @@ namespace GridTools
    * predicate is either an object of a type that has an <tt>operator()</tt>,
    * or it is a pointer to a non-member function, or it is a lambda function
    * object. In either case, argument and return
-   * value have to be of type Point<spacedim>.
+   * value have to be of type Point<spacedim>. An example of a simple
+   * transformation -- one that simply moves the object two units to the
+   * right -- could like as follows:
```

Comment:
```suggestion
   * transformation -- one that simply moves the object two units in
   * $x$-direction -- could look like this:
```
I would also specify the coordinate direction. And something's off in the main sentence.